docs/network-upgrades.md

```diff
@@ -12,17 +12,18 @@ import DraftExpectationsPartial from '@site/docs/partials/_draft-expectations-pa
 
 This page has information on the state and timelines of major software updates to Arbitrum One and Arbitrum Nova. Software updates that alter an Arbitrum chain's ability to produce valid Arbitrum blocks are referred to as ArbOS upgrades and, as outlined in the [Arbitrum DAO Constitution](./dao-constitution.md), will always require a Constiutional AIP to pass for DAO-governed chains.
 
-Visit [Inside Arbitrum Nitro](https://docs.arbitrum.io/inside-arbitrum-nitro/) to learn more about Nitro's architecture; more information about ArbOS software releases is available on the [Arbitrum DAO forum](https://forum.arbitrum.foundation/t/arbitrum-arbos-upgrades/19695).
+Visit [Inside Arbitrum Nitro](https://docs.arbitrum.io/inside-arbitrum-nitro/) to learn more about Nitro's architecture; more information about ArbOS software releases is available on the [Arbitrum DAO forum](https://forum.arbitrum.foundation/t/arbitrum-arbos-upgrades/19695) as well as in the [documentation for running Arbitrum nodes](https://docs.arbitrum.io/node-running/reference/arbos-software-releases/overview).
```

Comment:
non-blocking nit - using the title of the document in the link text can help set more precise expectations for readers. Not needed, and certainly not normalized across our doc sets, but an option to consider when linking docs to docs.

This target document doesn't have a clear title, which I've addressed via https://github.com/OffchainLabs/arbitrum-docs/pull/1030, which also includes updated style guidelines to make this guidance explicit.